packages/core/test/invoke.test.ts

```diff
@@ -2924,6 +2888,47 @@ describe('invoke', () => {
       .start();
   });
 
+  it.each([(ctx) => ({ endpoint: ctx.url }), { endpoint: (ctx) => ctx.url }])(
```

Comment:
imho it would be better to just split this into 2 explicit test cases, separate cases are easier to read and debug if needed